tests/unit/dataactvalidator/test_scripts_loadTas.py

```diff
@@ -73,24 +75,43 @@ def test_updateTASLookups(database, monkeypatch):
         }))
     )
 
-    sess.add(TASFactory(tas_id=222))
-    sess.add(TASFactory(tas_id=444, agency_identifier='other'))
+    sess.add_all([
+        # TAS present in both csv and db
+        TASFactory(account_num=222,
+                   **{field: '1' for field in loadTas._MATCH_FIELDS}),
+        # Example of TAS being modified
+        TASFactory(account_num=333, agency_identifier='old'),
+        # Example of TAS which is unrelated to anything of these entries
+        TASFactory(account_num=444, agency_identifier='old')
+    ])
     sess.commit()
 
-    results = sess.query(TASLookup).order_by(TASLookup.tas_id).all()
-    assert len(results) == 2
+    assert sess.query(TASLookup).count() == 3
     sess.invalidate()
 
     loadTas.updateTASLookups('file-name-ignored-due-to-mock')
 
-    results = sess.query(TASLookup).order_by(TASLookup.tas_id).all()
-    assert len(results) == 3    # there is no 444
-    assert results[0].tas_id == 111
-    assert results[0].agency_identifier == '0'
-    assert results[1].tas_id == 222
-    assert results[1].agency_identifier == '1'  # replaces previous value
-    assert results[2].tas_id == 333
-    assert results[2].agency_identifier == '2'
+    sess.invalidate()
+    results = sess.query(TASLookup).\
+        order_by(TASLookup.account_num, TASLookup.agency_identifier).all()
+    assert len(results) == 5
+    t111, t222, t333_new, t333_old, t444 = results
+
+    assert t111.account_num == 111
+    assert t111.internal_end_date == None               # still active
```

Comment:
Making sure I'm following along here...is `t111` `# still active` or is it an entirely new, active TAS? If it's the latter, does it make sense to tweak the comment to differentiate this from the the `t222` case? (something like `# new, active TAS`).